.buildkite/pipeline.yml

```diff
@@ -25,14 +25,15 @@ steps:
      - ./rpm-packages/*
    branches: "!master"
  - commands:
-   - nix-build ./nix -A binaries -o binaries
-   - ./scripts/test-binaries.sh binaries/bin
+   - buildkite-agent artifact download "binaries/bin/*" . --step "build and package"
+   - chmod +x ./binaries/bin/*
+   - nix-build tests/tezos-binaries.nix --no-out-link --arg path-to-binaries ./binaries/bin
    label: test binaries
    branches: "!master"
  - commands:
    - cd docker
    - nix run nixpkgs.docker -c ./docker-static-build.sh
-   - cd .. && ./scripts/test-binaries.sh docker
+   - cd .. && nix-build tests/tezos-binaries.nix --no-out-link --arg path-to-binaries ./docker
```

Comment:
Do we test binaries produced by nix AND binaries produced by docker?

Reply:
yes

Reply:
in different steps